apps/frontend/src/views/Results.vue

```diff
@@ -362,6 +355,12 @@ export default class Results extends mixins(RouteMixin) {
     this.search_term = '';
   }
 
+  mounted() {
+    if(this.serverMode) {
+      EvaluationModule.getAllEvaluations()
+    }
+  }
```

Comment:
This shouldn't be necessary if the evaluation already has the tags included when it is loaded.